README.md

```diff
@@ -19,18 +19,21 @@ To run the **HR Management** Web App, ensure your system meets the following req
 Follow the steps below to setup the project locally:
 
 * clone the repository: 
-git clone https://github.com/harry-olabs/avi-hub.git
+git clone 
+
+`https://github.com/harry-olabs/avi-hub.git`
+
 * Navigate to the project directory: 
 cd hr-management-web-app
 
 ## Install dependencies using npm 
 
-* *npm* install or *yarn* install
+* `npm install` or `yarn install`
```

Comment:
For simplicity, let's stick with npm. Remove `yarn` commands and make this a block code format